app/decorators/controllers/accounts_controller_decorator.rb

```diff
@@ -139,6 +158,49 @@ def get_last_assignment_due
     @last_assignment_due = SettingsService.get_settings(object: 'school', id: 1)['last_assignment_due']
   end
 
+  def get_course_start_time_hour
+    @course_start_time_hour = SettingsService.get_settings(object: 'school', id: 1)['course_start_time_hour'] || "12"
+  end
+
+  def get_course_start_time_minute
+    @course_start_time_minute = SettingsService.get_settings(object: 'school', id: 1)['course_start_time_minute'] || "00"
+  end
+
+  def get_course_time_ampm
+    @course_time_ampm = SettingsService.get_settings(object: 'school', id: 1)['course_time_ampm'] || "AM"
+  end
+
+  def set_course_time_ampm
```

Comment:
I understand that the hour, minute, and AM/PM fields are three distinct UI elements on the setttings page, but is there not a way to combine all three into one value/setting rather than making 3 set/get calls every time the page is viewed or the setting updated? Not sure what the technical hurdles would be to accomplish getting it down to one call instead...

Reply:
We have merged the three settings into one & got rid of those old getters/setters.